zbus/tests/basic.rs

```diff
@@ -112,6 +112,15 @@ fn fdpass_systemd() {
     f.metadata().unwrap();
 }
 
+#[cfg(target_os = "linux")]
+#[cfg(feature = "ibus")]
+#[test]
+fn test_ibus() {
+    let ibus = block_on(conn::Builder::ibus().unwrap().build()).unwrap();
+
+    assert!(!ibus.server_guid().as_str().is_empty());
```

Comment:
if we're going to check the guid, we should do a bit better job than just checking if it's empty. :) You can issue the command here to re-fetch the guid and then compare the guids.